### PR TITLE
[fix-4945]: add correct icons to filters Incident Map

### PR DESCRIPTION
**Pull request by vdegraaf**
Ticket: [SIG-4945](https://datapunt.atlassian.net/browse/SIG-4945)

[SIG-4945]: https://datapunt.atlassian.net/browse/SIG-4945?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/signals/IncidentMap/components/utils/get-filtered-incidents.test.ts [hunk 1]
```diff
@@ -1,13 +1,65 @@
 // SPDX-License-Identifier: MPL-2.0
 // Copyright (C) 2022 - 2023 Gemeente Amsterdam
 import { mockFiltersShort } from '../__test__/mock-filters'
-import { mockIncidentsShort } from '../__test__/mock-incidents'
+import { mockIncidentsWithoutIcon } from '../__test__/mock-incidents-without-icon'
 import { getFilteredIncidents } from './get-filtered-incidents'
 
+const mockFiltersAllActive = mockFiltersShort.map((filter) => {
+  if (filter.subCategories) {
```
Comment (OscarBakker) [inline src/signals/IncidentMap/components/utils/get-filtered-incidents.test.ts:8]: I would check on Array.isArray(filter.subCategories). Seems a little effort.

Comment (OscarBakker) [reply src/signals/IncidentMap/components/utils/get-filtered-incidents.test.ts:8]: or is it typed?

File: src/signals/IncidentMap/components/utils/get-filtered-incidents.test.ts [hunk 2]
```diff
@@ -1,13 +1,65 @@
...
+    return {
+      ...filter,
+      filterActive: true,
+      subCategories: filter.subCategories.map((subFilter) => {
```
Comment (OscarBakker) [inline src/signals/IncidentMap/components/utils/get-filtered-incidents.test.ts:12]: subcategory is actually one word and camelcase should only be used to substitute spaces, but maybe too much of a hassle.

**Review by OscarBakker — COMMENTED** [review]
some comments

File: src/signals/IncidentMap/components/utils/get-filtered-incidents.test.ts (hunk 2 above)
Comment (vdegraaf) [reply src/signals/IncidentMap/components/utils/get-filtered-incidents.test.ts:12]: These are typed indeed.

File: src/signals/IncidentMap/components/utils/get-filtered-incidents.test.ts (hunk 1 above)
Comment (vdegraaf) [reply src/signals/IncidentMap/components/utils/get-filtered-incidents.test.ts:8]: it is typed indeed

**Review by OscarBakker — APPROVED** [review]
good!

**Pull request merged**